src/cc/makerogue

```diff
@@ -19,7 +21,7 @@ if [ "$HOST" = "x86_64-w64-mingw32" ]; then
   fi
 else
   echo building rogue...
-  ./configure
+  ./configure --build=aarch64-unknown-linux-gnu
```

Comment:
This is an upstream KMD bug we should probably fix and send upstream. Would `./configure --build=$HOST` work?